### PR TITLE
Add triggers v0.11.2, v0.12.1, v0.13.0 and v0.14.1

### PR DESCRIPTION
**Pull request by eliasnorrby**
# Changes

The website lists v0.10.1 as the latest release of triggers. This PR adds v0.11.2, v0.12.1, v0.13.0 to the sync config.

# Submitter Checklist

These are the criteria that every PR should meet, please check them off as you
review them:

- [x] Includes [docs](https://github.com/tektoncd/community/blob/master/standards.md#principles) (if user facing)
- [x] Commit messages follow [commit message best practices](https://github.com/tektoncd/community/blob/master/standards.md#commit-messages)

### CODE REVIEW

File: sync/config/triggers.yaml [hunk 1]
```diff
@@ -30,6 +30,48 @@ archive: https://github.com/tektoncd/triggers/tags
 #       header: <dict>         # optional, no header added if not set
 #         See https://www.docsy.dev/docs/adding-content/navigation/#section-menu
 tags:
+- name: v0.13.0
```
Comment (afrittoli) [inline sync/config/triggers.yaml:33]: NIT: could you keep the list indentation consistent with the rest of the config?

**Comment by afrittoli** [comment]
Thanks for this!
I'm not sure why - the link title in the preview is not correct https://deploy-preview-270--tekton.netlify.app/docs/triggers/ 🤔

File: sync/config/triggers.yaml (hunk 1 above)
Comment (eliasnorrby) [reply sync/config/triggers.yaml:33]: Sure thing! But I think this is consistent? The rest of the config is not indented either.

**Comment by eliasnorrby** [comment]
> Thanks for this!
> I'm not sure why - the link title in the preview is not correct https://deploy-preview-270--tekton.netlify.app/docs/triggers/

Yeah, I noticed too. It's because of the frontmatter in [triggers/docs/README.md](https://github.com/tektoncd/triggers/blob/main/docs/README.md). I was about to open a PR there too, to update the `title` and `linkTitle` fields to better match the context.

File: sync/config/triggers.yaml (hunk 1 above)
Comment (afrittoli) [reply sync/config/triggers.yaml:33]: Oh, never mind, it looks like it's the way github displays it that made it look different? Sorry about that...

**Comment by afrittoli** [comment]
> Yeah, I noticed too. It's because of the frontmatter in [triggers/docs/README.md](https://github.com/tektoncd/triggers/blob/main/docs/README.md). I was about to open a PR there too, to update the `title` and `linkTitle` fields to better match the context.

Yeah, indeed. You'll need PRs to the various branches though. 
The other issue I see is that the config points to tags, so either we make new tags on triggers side, or alternatively we could try and use branches as an alternative to tags in the website config.

**Comment by eliasnorrby** [comment]
Right, so to make the archive work we would either need to create new releases/tags (e.g. v0.11.3) or use branches for references. There seems to be release branches available for v0.11 and v0.12 (e.g. `release-v0.11.x`), but not for v0.13 (or v0.14 that was just released, for that matter). 

Is a documentation update like this enough to warrant a new patch release? If the `sync.py` script works using a branch ref in place of a tag, maybe that's a better approach. i.e.

```yaml
tags:
- name: release-v0.13.x
  displayName: v0.13.0
```

**Comment by eliasnorrby** [comment]
A few other questions/considerations:

**TL;DR:** Should we just remove the `docs/getting-started` entry from `folders`, or try to make it work?

In [a comment](https://github.com/tektoncd/website/issues/34#issuecomment-744738016) on #34, you mention a script for updating the config. I couldn't find this script and therefore just copied the block for release v0.10.1. I hope this is fine.

But that leads me to another question: the `docs/getting-started` item under `folders` seems to be ineffectual? I just copied the block for v0.10.1, but I don't see these files showing up anywhere on the website. 

The block looks a little different for all the previous releases, e.g.

```yaml
- name: v0.10.1
  # ...
  folders:
    # ...
    docs/getting-started:
      target: getting-started
      include:
      - create-ingress.yaml
      - create-webhook.yaml
```
vs
```yaml
- name: v0.6.1
  # ...
  folders:
    # ...
    docs/getting-started:
      target: ''
      include:
      - create-ingress.yaml
      - create-webhook.yaml

```

It seems these different variants were introduced in [this refactoring](https://github.com/eliasnorrby/website/commit/85a5bcb687a8d3ad35946033c6fff41e029ee70b#diff-043f26172907c43017b56d84eca8d894137f5d84573a67838bb6fc2059a64b7e)

The way I read it, the intention of the block seems to be to map the contents of https://github.com/tektoncd/triggers/blob/main/docs/getting-started to https://tekton.dev/docs/getting-started (is my understanding correct?), but that's not what's happening. As is, none of the markdown documents are included, only the two yaml-files.

https://tekton.dev/docs/getting-started now holds a "Getting started with Pipelines" guide, but those files are hosted in `tektoncd/website`, not in `tektoncd/pipelines`, so it seems to be in a bit of a weird state.

Also, assuming my understadning of the `target` property is correct, using it in archived releases doesn't make much sense, because when viewing documentation for a previous release, the sidebar navigation is limited to the tree for the selected project. So, when viewing the docs for e.g. triggers@v0.10.1, it's not possible to get to the Getting Started section at all.

**Comment by afrittoli** [comment]
> A few other questions/considerations:
> 
> **TL;DR:** Should we just remove the `docs/getting-started` entry from `folders`, or try to make it work?
> 
> In [a comment](https://github.com/tektoncd/website/issues/34#issuecomment-744738016) on #34, you mention a script for updating the config. I couldn't find this script and therefore just copied the block for release v0.10.1. I hope this is fine.

This is the script: https://github.com/tektoncd/website/blob/main/sync/README.md#versions - but manual copy is absolutely fine.

> 
> But that leads me to another question: the `docs/getting-started` item under `folders` seems to be ineffectual? I just copied the block for v0.10.1, but I don't see these files showing up anywhere on the website.
> 
> The block looks a little different for all the previous releases, e.g.
> 
> ```yaml
> - name: v0.10.1
>   # ...
>   folders:
>     # ...
>     docs/getting-started:
>       target: getting-started
>       include:
>       - create-ingress.yaml
>       - create-webhook.yaml
> ```
> 
> vs
> 
> ```yaml
> - name: v0.6.1
>   # ...
>   folders:
>     # ...
>     docs/getting-started:
>       target: ''
>       include:
>       - create-ingress.yaml
>       - create-webhook.yaml
> ```
> 
> It seems these different variants were introduced in [this refactoring](https://github.com/tektoncd/website/commit/85a5bcb687a8d3ad35946033c6fff41e029ee70b#diff-043f26172907c43017b56d84eca8d894137f5d84573a67838bb6fc2059a64b7e)
> 
> The way I read it, the intention of the block seems to be to map the contents of https://github.com/tektoncd/triggers/blob/main/docs/getting-started to https://tekton.dev/docs/getting-started (is my understanding correct?), but that's not what's happening. As is, none of the markdown documents are included, only the two yaml-files.

No, that's not the intention. The purpose of that block is to have working links to those two yaml files.
Look at https://tekton.dev/docs/triggers/exposing-eventlisteners/, the `create-ingress` link points to tekton.dev and it allows you to download the file.
I'm not sure if the `create-webhook` YAML is linked to from anywhere anymore?

> 
> https://tekton.dev/docs/getting-started now holds a "Getting started with Pipelines" guide, but those files are hosted in `tektoncd/website`, not in `tektoncd/pipelines`, so it seems to be in a bit of a weird state.
> 
> Also, assuming my understadning of the `target` property is correct, using it in archived releases doesn't make much sense, because when viewing documentation for a previous release, the sidebar navigation is limited to the tree for the selected project. So, when viewing the docs for e.g. triggers@v0.10.1, it's not possible to get to the Getting Started section at all.

**Comment by eliasnorrby** [comment]
> No, that's not the intention. The purpose of that block is to have working links to those two yaml files.
> Look at https://tekton.dev/docs/triggers/exposing-eventlisteners/, the create-ingress link points to tekton.dev and it allows you to download the file.

I see, that makes sense!

**Comment by eliasnorrby** [comment]
> If the sync.py script works using a branch ref in place of a tag, maybe that's a better approach. i.e.

[This looks promising](https://github.com/tektoncd/website/blob/e8b68d6b5eb23675decf98c2d6e00e4187fb3947/sync/sync.py#L108-L117)

I'll experiment a little.

File: sync/config/triggers.yaml [hunk 2]
```diff
@@ -30,6 +30,58 @@ archive: https://github.com/tektoncd/triggers/tags
...
+      index: README.md
+      include: ['*.md']
+    docs/getting-started:
+      target: getting-started
+      include:
+      - create-ingress.yaml
+- name: origin/release-v0.12.x
+  # The name to display on tekton.dev.
+  # sync.py will use this value in the version switcher and other places.
+  displayName: v0.12.1
```
Comment (eliasnorrby) [inline sync/config/triggers.yaml:62]: @afrittoli As far as I can tell, this seems to work. This would allow for patching the frontmatter of the relevant release branches within `tektoncd/triggers`, without adding new tags. What do you think?

**Comment by eliasnorrby** [comment]
Progress for fixing link titles:
- [x] `main` MERGED tektoncd/triggers#1095
- [x] `release-v0.14.x` MERGED tektoncd/triggers#1109
- [ ] `release-v0.13.x` No branch available
- [ ] `release-v0.12.x` OPEN tektoncd/triggers#1111

@afrittoli, can you push the `release-v0.13.x` branch to the `v0.13.0` tag?

**Comment by eliasnorrby** [comment]
Alright, now that all the the `triggers` PRs have been merged, I've updated this PR by pointing to remote references instead of tags. Deploy preview is up and looks good to me. 

Something to keep in mind: when using a remote ref (i.e. `origin/release-v0.14.x`), the docs displayed can become out of sync with the displayed name. 

``` yaml
- name: origin/release-v0.12.x
  displayName: v0.12.1
```

If commits are pushed to `release-v0.12.x`, the docs will automatically update to reflect any changes. Notably, if/when release `v0.12.2` is published, the documentation content will update, but the version select menu item will still display `v0.12.1`, not `v0.12.2`.

**Review by afrittoli — COMMENTED** [review]
Thank you so much for this!
/approve

**Comment by afrittoli** [comment]
@tektoncd/triggers-maintainers FYI
